ReactAndroid/src/main/java/com/facebook/react/views/swiperefresh/ReactSwipeRefreshLayout.java

```diff
@@ -20,11 +20,26 @@
  */
 public class ReactSwipeRefreshLayout extends SwipeRefreshLayout {
 
+  private boolean mRefreshing = false;
+
   public ReactSwipeRefreshLayout(ReactContext reactContext) {
     super(reactContext);
   }
 
   @Override
+  public void setRefreshing(boolean refreshing) {
+    mRefreshing = refreshing;
+    // Use `post` otherwise the control won't start refreshing if refreshing is true when
+    // the component gets mounted.
+    post(new Runnable() {
```

Comment:
Yeah, that other runnable was added pretty recently and the same bug could happen to that value too.

I'll have another look at this tonight see if I can make this cleaner and make it consistent.